content/en/docs/kitex/Tutorials/advanced-feature/start_shutdown_hook.md

```diff
@@ -1,8 +1,70 @@
 ---
 title: "Customize Hook for Start/Shutdown of Server-side"
-date: 2022-07-20
+date: 2022-07-23
 weight: 5
 description: >
 ---
 
-Translation TODO
+Kitex provides a global Hook for injecting your processing logic on the server-side **after** triggering **startup** and **before exiting**.
+
+Also, you can modify the main function to customize some processing logic **before** the server-side **starts** and **after** it **exits** (listening for port closures).
+
+## After the server-side start and before exit
+
+Since the server-side handles logic inside the framework after startup and before exit, users who want to customize the business logic need to use Hook.
```

Comment:
Since the server-side logic after startup and before exit is processed inside the framework, users who want to customize the business logic need to use Hook for injection.
- > Think about that